extensions/cornerstone/src/services/SegmentationService/SegmentationService.ts

```diff
@@ -105,7 +109,7 @@ class SegmentationService {
     toolGroupId?: string,
     properties?: {
       label?: string;
-      color?: Types.Point3;
+      color?: csTypes.Point3;
```

Comment:
Lets get an actual color type in OHIF core types rather than assuming we are using the cornerstone one.